website/public/js/controllers/partyCtrl.js

```diff
@@ -51,6 +51,11 @@ habitrpg.controller("PartyCtrl", ['$rootScope','$scope','Groups','Chat','User','
           Analytics.track({'hitType':'event','eventCategory':'button','eventAction':'click','eventLabel':'Leave Party'});
           $scope.selectedGroup = group;
           $scope.popoverEl = $($event.target);
+
+          if ($scope.popoverEl[0].localName == "span") {
+              $scope.popoverEl = $($scope.popoverEl[0].parentNode);
+          }
```

Comment:
I don't think checking the `.btn` class will work in this instance, since the span portion does not have a button class. I agree that this isn't exactly the correct solution, but it's on the right track. I'll need to investigate this a little and see if there's a more meaningful way of accomplishing this. Especially in a way that won't break if we change that span to a div :)

One other thing. Do we need to do the same thing in the guild controller?